polyfill/test/validStrings.mjs

```diff
@@ -293,20 +293,11 @@ const timeZoneNumericUTCOffsetNotAmbiguousAllowedNegativeHour = withCode(
   saveOffset
 );
 const timeZoneUTCOffset = choice(utcDesignator, timeZoneNumericUTCOffset);
-const timeZoneUTCOffsetName = seq(
-  sign,
-  hour,
-  choice([minuteSecond, [minuteSecond, [fraction]]], seq(':', minuteSecond, [':', minuteSecond, [fraction]]))
-);
-const timeZoneBracketedName = withCode(
-  choice(timeZoneUTCOffsetName, ...timezoneNames),
```

Comment:
Am I correct in assuming the `Etc/GMT±n` names are included in `timezoneNames` so we don't have to model them separately here?

Reply:
right, that's my intention

Reply:
Or at least, my intention was to remove definitions in validStrings.mjs that have been removed from the grammar (the aim being to make sure that those the two mirror each other as closely as possible).